src/components/DocumentationTopic/TopicLinkBlockIcon.vue

```diff
@@ -41,7 +39,7 @@ const TopicRoleIcons = {
 };
 
 export default {
-  components: { OverridableAsset, SVGIcon },
+  components: { SVGIcon },
   props: {
```

Comment:
I think we could remove the `imageOverride` prop here and no need to pass it in the `TopicsLinkBlock` component.